rest-api/src/main/resources/definitions/request_params.yml

```diff
@@ -0,0 +1,46 @@
+type: object
+description: |
+    Type-safe object that returns request parameters submitted for a request. Only the values that 
+    are submitted in a request will be returned, the rest will be null.
+properties:
+    pageSize:
+        type: integer
+        format: int32
+    total:
+        type: integer
+        format: int32
```

Comment:
Minor: Can you sort these in alphabetical order to make it easier to find attributes?